dpanonymize/__init__.py

```diff
@@ -89,7 +99,7 @@ def get_file_objects_from_module(root_dir: Union[Path, str],
         - module: name of the module to remove PII from, str.
         - BIDS: True if the PHOENIX is in BIDS structure, bool.
     '''
-    module_dirs = list(root_dir.glob(f'*/{module}')) if BIDS else \
+    module_dirs = list(root_dir.glob(f'*/*/*/{module}')) if BIDS else \
         list(root_dir.glob(f'*/*/{module}'))
```

Comment:
Please write two example `path/to/{module}` for each `BIDS` so we are confident about the glob patterns. I remember you have had a slide, that might be helpful alone too.

Reply:
![image](https://user-images.githubusercontent.com/3075783/129820684-a115fdaf-7f08-40f6-8cd5-0dc8b1e8607a.png)

I see where the confusion is coming from. I'll double-check this part in a test module and get back to this thread.

Reply:
Below are the examples for each `BIDS` bools. `dpanonymize` works with both `BIDS` bool structures, and this is tested by `pytest -s dpanonymize/tests/dpanonymize/test_dpanonymize.py`

BIDS = True
```
# actigraphy csv
tmp_phoenix/GENERAL/BWH/subject01/actigraphy/raw/subject01.BWH.csv

# mri dicom
tmp_phoenix/GENERAL/BWH/subject01/mri/raw/subject01.BWH.dcm
```

BIDS = False
```
# actigraphy csv
tmp_phoenix/GENERAL/BWH/raw/subject01/actigraphy/subject01.BWH.csv

# mri dicom
tmp_phoenix/GENERAL/BWH/raw/subject01/mri/subject01.BWH.dcm
```

Reply:
Thanks Tashrif - the confusing doc strings for the `get_file_objects_from_phoenix` and `get_file_objects_from_module` function has been updated to clearly state the input is the PROTECTED path of the phoenix structure. 
f146f71